crates/iota/src/iota_commands.rs

```diff
@@ -206,10 +206,10 @@ pub enum IotaCommand {
         /// Defaults to `200000000000`(200 IOTA).
         #[arg(long)]
         faucet_amount: Option<u64>,
-        /// Set the amount of objects the faucet will send for each request.
-        /// Defaults to 5.
+        /// Set the amount of coin objects the faucet will send for each
+        /// request. Defaults to 5.
         #[arg(long)]
-        faucet_object_count: Option<usize>,
+        faucet_coin_count: Option<usize>,
```

Comment:
Sorry to do this again but maybe even better would be `faucet_coins_per_request`?

Reply:
I think I would rather keep it as `faucet_coin_count` because it's shorter and more similar to `faucet_amount`
The `_per_request` detail is explained in the comment for it